rootfs/bin/is_running

```diff
@@ -4,4 +4,4 @@
 set -e
 
 # check if database is running
-gosu postgres pg_ctl status
+gosu postgres psql -l -t >/dev/null 2>&1
```

Comment:
The line

```
if [[ -f "$PGDATA/recovery.conf" ]]; then
  # We don't need to call psql for checking in the recovery mode.
  # As WAL-E calls psql internally.
  gosu postgres pg_ctl status
else
[...]
```

Will mean that `/bin/is_running` will succeed when the database is in recovery, which is what we want to avoid as it is not ready to accept connections. `pg_ctl status` returns 0 when the database is booted but in recovery mode. We only want to return success when the database is truly ready to accept connections, which is why I'm just calling `psql -l -t`.

For reference, this is how we used to check readiness [in v1](https://github.com/deis/deis/blob/master/database/bin/boot#L121-L122) and we had no issues running it in production.